src/configuration.js

```diff
@@ -150,6 +151,7 @@ export default class Configuration {
      * @param {Object} [options.logger] - a logger (see ./_flow/logger.js)
      * @param {Object} [options.tags] - set of key-value pairs which will be set
      *        as process-level tags on the Tracer itself.
+     * @param {Object} [options.upsampleChildSpans] - if set, the tracer will upsample child spans
```

Comment:
I think we should have a top-level "upsampling" flag that refers to first-in-process span, in addition to `upsampleChildSpans`. It could be a hierarchical struct like
```javascript
'upsampling': {
  'enabled': true,
  'allowOnInnerSpans': false,
}
```

Because without it we're changing the existing behavior of the client where it respects the upstream sampling decision.